commands/hugo.go

```diff
@@ -593,8 +605,15 @@ func copyStatic() error {
 // getDirList provides NewWatcher() with a list of directories to watch for changes.
 func getDirList() []string {
 	var a []string
+	var layoutPath string
+
+	if layoutDir != "" {
+		layoutPath = helpers.AbsPathifySimple(viper.GetString("LayoutDir"))
```

Comment:
If the working directory is set, this allows relative layout/content path to be set independently and working.
Perhaps this feature is not needed yet or that this func needs to be renamed?